### DISCUSSION TITLE
Duplicated accounts by error

### DISCUSSION THREAD

**Discussion by alainux** [vercel/community#7095] [Account Issue] [answered]
### Question

I signed up for a Pro plan with a new email which created a new account and team, and then when I connected it to Github, all of the deployments from my other (my main) account and teams stopped working. I fixed it by re-authenticating Github, but I need the new account to also work, though. 

How can I "merge" both accounts?

**Comment by amyegan** [comment, marked as answer]
Hey @alainux. A GitHub account can only be connected to one Vercel team at a time, as you discovered.

For security reasons we are not able to merge accounts on our end. You could either [transfer team ownership](https://vercel.com/guides/how-do-i-transfer-ownership-of-a-vercel-team) or [move all projects](https://vercel.com/docs/projects/overview#transferring-a-project) to your preferred account and delete the empty account. 

If you have more questions or need help with the transfer process, please [submit a case](https://vercel.com/help) to the Customer Success team. They'll be able to discuss more account details with you.
